lib/assets/us_county_and_equivalent_list.csv

```diff
@@ -3198,4 +3198,44 @@
 "Zapata County,  Texas", Texas
 "Zavala County,  Texas", Texas
 "Ziebach County,  South Dakota", South Dakota
-"Other US County,  other US region", other US region
\ No newline at end of file
+"Other US County,  other US region", other US region
```

Comment:
Actually, this probably shouldn't be a County in the database. We should have separate logic to ensure this always comes last. Otherwise it'll be in a weird middle-of-the-list. Technically we can just move this line to the end of this CSV, but my hunch is there'll be some ordering tomfoolery which will make it not always come last.

Reply:
Hmm.  I see what you're saying -- though with a list this long most (I'm not going to be putting a number on it) people are going to be accessing their counties by typing ahead, so I'm not sure it's that big an issue.   

Honestly, I had not noticed that I wasn't ordering alpha before displaying -- probably because the original list was in strict alpha.   I'm going to fix that.

I also thought about whether we could just remove "Other US county"  if we have total US coverage.  But we might have a case where someone has 2% spread over several counties,  and would want to just use it for that.

Reply:
And moving it to the end wouldn't work -- it's already in the db, after all, so if it's ordered by creation, it would be weirdly at the end, and if it's alpha, it will be in the middle.

Reply:
We *could*  add an additional ordering priority,  set everything else to something low, and Other to 9999, and then order by alpha within the ordering value .     I think it's a mite kludgey, but if keeping Other at the end is important, that might be the easiest way.

Reply:
@dorner -- What do you think of that?   It would also have the advantage of allowing us to put all the US counties first if we wanted to if/when we add other countries (and we do have a Toronto bank that has signed up)

Reply:
I don't think that makes a lot of sense... maybe add a type/metadata/category etc. column which we can use to group results. Then you can do a sort by category + name. And if Other has its own category then we can ensure it shows up the right way.

Reply:
Ok.  Allows the same results-ish but with more sophistication.  Assuming support doesn't blow up I'll try to get that in tomorrow.

Reply:
Also noticed I had a leading space on the regions.  Fixed that.